themes/navy/source/js/main.js

```diff
@@ -410,6 +410,25 @@ $(document).ready(function($) {
     else header.css(relative);
   });
 
+  // Unfold an FAQ item based on ID
+  if (document.title === 'Status - FAQ' && window.location.hash.length) {
```

Comment:
I think checking path is better than checking title.
```suggestion
  if (window.location.pathname === '/faq/') {
```
Also, we don't want to depend on the hash being set if we are also handling clicks.